src/api/app/models/project.rb

```diff
@@ -1240,6 +1240,16 @@ def open_requests
     { reviews: reviews, targets: targets, incidents: incidents, maintenance_release: maintenance_release }
   end
 
+  def open_incoming_requests
+    ids = open_requests.values.sum
+    BsRequest.includes(:bs_request_actions).where(id: ids).where(bs_request_actions: { target_project: name })
```

Comment:
[open_requests](https://github.com/openSUSE/open-build-service/blob/260ba3593885feecb54c8e9b55cd83002982bf86/src/api/app/models/project.rb#L1212) doesn't return request IDs, but request numbers. This line should be changed. Same for the other in `open_outgoing_requests`.

Reply:
Good catch, thank you. Fixed.